src/pages/TokenPage/index.tsx

```diff
@@ -0,0 +1,260 @@
+import React from 'react';
+
+import { useAllTokens, Token } from 'queries';
+import { Helmet } from 'react-helmet-async';
+import { Link } from 'react-router-dom';
+import { Box, Flex, Text } from 'rebass/styled-components';
+import styled from 'styled-components';
+
+import Footer from 'components/Footer';
+import Header from 'components/Header';
+import { CurrencyKey } from 'constants/currency';
+import { tokenInfo } from 'constants/tokenInfo';
+import { getCurrencyKeyIcon } from 'utils';
+import { getFormattedNumber } from 'utils/formatter';
+
+const Container = styled(Box)`
```

Comment:
I can see some same components(`Container`, `CurrencyIcon`,..) used in the codebase. Please DRY the code as you can do.